src/plugins/intel_cpu/src/nodes/space_to_batch.cpp

```diff
@@ -112,6 +105,22 @@ static std::vector<size_t> getShape5D(const SizeVector &shape) {
 
 template<typename T>
 void SpaceToBatch::SpaceToBatchKernel() {
+    const auto *blockShapesPtr = reinterpret_cast<int *>(getParentEdgeAt(1)->getMemoryPtr()->GetPtr());
+    size_t typeSize = dnnl::memory::data_type_size(getParentEdgeAt(1)->getMemory().GetDataType());
```

Comment:
Does this op support dynamic rank ? If not, then the `N` could be got from the input rank.
If we must use `typesize` to calculate, then it's better not to get it with `getOriginalInputPrecisionAtPort(1).size()`, I think the precision is fixed during compilation phase.

Reply:
In current cases rank is fixed. It's sufficient to use data rank. Comment applied.